src/azure/cli/_help.py

```diff
@@ -386,5 +386,13 @@ def _get_single_metadata(cmd_table):
     assert len(cmd_table) == 1
     return next(metadata for _, metadata in cmd_table.items())
 
+def _get_group_priority(group_name):
+    priorities = {
+        'Resource Id Arguments': 10,
+        'Global Arguments': 1000,
+        }
```

Comment:
Very much liking the addition of group priorities! Is there where I'd have to add them? Seems a little wonky, but it could be good enough. There aren't more than a handful of groups anyways (my storage PR introduces a few more for example).

If a group isn't in the list does it default to alphabetical? If so that'd probably be perfect.